suricata.yaml.in

```diff
@@ -80,7 +80,8 @@ stats:
 
 # Plugins -- Experimental -- specify the filename for each plugin shared object
 plugins:
-#   - /path/to/plugin.so
+  @pfring_comment@- @prefix@/lib/@PACKAGE_NAME@/pfring.so
```

Comment:
If pfring was built, this will be uncommented by default. But we might want the user to opt-in and uncomment themselves - the path should be correct tho.

Can cause fail to crash if `libpfring` hasn't been `ldconfig'd` yet, but not much we can do about that.